src/style.css

```
@@ -1 +1,91 @@
-/* add styles here */
+@font-face {
+  font-family: 'Avenir M';
+  src:
+    local('Avemir'),
+    url(fonts/Avenir-Book.ttf),
+    url(fonts/Avenir-Heavy.ttf) format(avenir);
+}
+
+body {
+  margin: 0;
+  padding-top: 20px;
+}
+
+.form {
+  position: relative;
+  margin-right: 8px;
+  margin-left: 8px;
+}
+
+.form [class='big-input'] {
```

No need to use attributes as css selector, fix it everywhere
```suggestion
.big-input {
```